.travis.yml

```diff
@@ -58,6 +61,11 @@ script:
       amd64)
         GOARCH=amd64 ./test
         ;;
+      other-amd64)
```

Comment:
any reason why windows and darwin can't be separate targets? would make it clearer which build is failing